controllers/policyserver_controller_test.go

```diff
@@ -142,59 +146,333 @@ var _ = Describe("PolicyServer controller", func() {
 		})
 	})
 
-	When("creating a new PolicyServer", func() {
-		var policyServer *policiesv1.PolicyServer
+	It("should use the policy server affinity configuration in the policy server deployment", func() {
```

Comment:
can we group all the creation-related blocks into a `When("creating a PolicyServer")` ?